components/features/VideoConference/ConferenceWrapper/ConferenceWrapper.tsx

```diff
@@ -37,12 +40,22 @@ const ConferenceWrapper: React.FC<Props> = ({ token }) => {
     // eslint-disable-next-line
   }, [isConnected]);
 
+  const endRoom = async () => {
+    await actions.endRoom(false, "Room ended by host");
+    setRoomEnded(true);
+  };
+
+  if (roomEnded) {
+    router.push("/");
```

Comment:
Hello, rashad! I am curious to why we need to redirect back to the home page after we ended the room. I believe it's a 100ms thing, but I'm still curious anyways. Let me know 😀

Reply:
Hello Alex, yeah apparently the downside of 100ms is that their promises aren't actually real promises so I have to figure out a workaround and ended up settling with this :D